content/ri/memory-optimizations.md

```diff
@@ -70,157 +71,154 @@ You can follow one of these three steps to reclaim the memory:
 1. To reclaim the expired keys faster, increase memory-samples in redis.conf (default is 5).
 1. You can set up a cron job that runs the scan command after an interval which helps in reclaiming the memory of the expired keys.
 
-#### Trade offs
+#### Trade-offs
 
-If we increase the `maxmemory-samples` config, it expires the keys faster, but it costs more CPU cycles, which increases latency of commands. Secondly, increasing the expiry of keys helps but that requires significant changes to application logic.
+If you increase the `maxmemory-samples` configuration parameter, it expires the keys faster, but it costs more CPU cycles, which increases latency of commands. Secondly, increasing the expiry of keys helps but that requires significant changes to application logic.
 
 ### Use better serializer
 
-Redis does not have any specific data type to store the serialized objects, they are stored as byte array in Redis. If we are using regular means of serializing our java,python and PHP objects, they can be of larger size which impacts the memory consumption and latency.
+Redis does not have any specific data type to store the serialized objects, they are stored as byte array in Redis. If you are using regular means of serializing our Java, Python, and PHP objects, they can be of larger size which impacts the memory consumption and latency.
 
-#### Which serializers to use
+#### Choosing a serializer
 
-Instead of default serializer of your programming language (java serialzed objects, python pickle, PHP serialize etc), switch to a better library. There are various libraries like - Protocol Buffers, MessagePack etc.
+Instead of using the default serializer of your programming language (Java serialized objects, Python pickle, PHP serialize, and so on), switch to a better library like Protocol Buffers, MessagePack, and so on.
 
 ##### MessagePack
 
 MessagePack is an efficient binary serialization format. It lets you exchange data among multiple languages like JSON. But it's faster and smaller. Small integers are encoded into a single byte, and typical short strings require only one extra byte in addition to the strings themselves.
 
-As said by Salvatore Sanfilippo, creater of Redis
-
-`Redis scripting has support for MessagePack because it is a fast and compact serialization format with a simple to implement specification. I liked it so much that I implemented a MessagePack C extension for Lua just to include it into Redis.`
+As stated by Salvatore Sanfilippo, creator of Redis, "Redis scripting has support for MessagePack because it is a fast and compact serialization format with a simple to implement specification. I liked it so much that I implemented a MessagePack C extension for Lua just to include it into Redis.""
```

Comment:
```suggestion
As stated by Salvatore Sanfilippo, creator of Redis, "Redis scripting has support for MessagePack because it is a fast and compact serialization format with a simple to implement specification. I liked it so much that I implemented a MessagePack C extension for Lua just to include it into Redis."
```